pyscreeze/__init__.py

```diff
@@ -763,5 +963,6 @@ def pixel(x, y):
 locateAll = _locateAll_pillow
 if _useOpenCV:
     locateAll = _locateAll_opencv
-    if not RUNNING_PYTHON_2 and cv2.__version__ < '3':
+    # https://github.com/opencv/opencv/issues/23950
+    if not RUNNING_PYTHON_2 and cv2.__version__ < '3':  # type: ignore[attr-defined]
```

Comment:
Ref https://github.com/opencv/opencv/issues/23950

Reply:
```suggestion
    if not RUNNING_PYTHON_2 and cv2.__version__ < '3':
```
This issue has been resolved.
Btw @asweigart could I get a review and merge on this? Thanks.